covalent_dispatcher/_db/update.py

```diff
@@ -87,7 +88,10 @@ def _node(
 
     Returns:
         None
+
     """
+    if not node_name:
```

Comment:
If comparing with `None` my suggestion for the ideal way would be to do `if node_name is None`.